contracts/proxy/ImplementationAuthority.sol

```diff
@@ -14,6 +23,7 @@ contract ImplementationAuthority is IImplementationAuthority, Ownable {
     constructor(address _implementation) public {
         implementation = _implementation;
         emit UpdatedImplementation(_implementation);
+        owner = msg.sender;
```

Comment:
i have a problem with this... imo either you add the functions to transfer the ownership, which is my preferred solution, either you put the owner as a parameter of the constructor, because i am not sure that the deployment wallet will always be the owner, in fine, and at the moment, there is no way to change the owner address with the current implementation.